queries/gap/highlights.scm

```diff
@@ -0,0 +1,219 @@
+; NOTE: (reiniscirpons) in case multiple queries match, last query wins. So
+; queries should go from least specific to most specific. (This is the default
+; behaviour since tree-sitter 0.22.2)
+(identifier) @variable
+
+; Constants
+; convention: constants are of the form ALL_CAPS_AND_UNDERSCORES and have length at least 2
```

Comment:
We're trying to get away from this convention-based highlighting. It's not worth the headache, all things considered.